src/app/lib/views/browser/list.js

```diff
@@ -386,7 +394,7 @@
             if (index === -1) {
                 index = 0;
             } else {
-                index = index + NUM_ITEMS_IN_ROW;
+                index = index + this.itemsPerRow();
```

Comment:
can we merge the 2 last functions ? so we don't have, for instance, limit case working on Up but not on Down ?